common/live/prd/infra/development/terraform.tfvars

```diff
@@ -0,0 +1,19 @@
+# ↓ Module metadata
```

Comment:
@amatas Check this out:
https://github.com/natarajaya/exekube-test
I created test exekube project with proposed `live` structure and it works great! No issues with dependencies.
cc @mrtyler

Reply:
I think the innovation that makes Sergey's version work is moving the code that creates zone resources out of `development/terraform.tfvars` (i.e. the directory itself) and into a module `development/root/terraform.tfvars`. I like that part.

However, I'm not sure I like the name `root`. Next to entries like `tyler` and `natarajaya`, it makes me think of the root user, not DNS. How about keeping the dedicated directories, but calling them `zone` instead?

I like changing the name of top-level `prd` to something else to reflect the unusual/singleton-y nature of the common/ side of gpii-infra. I was going to say that I don't like `gpii-net` since there are (or will be) more than DNS resources managed by this project, but I can't think of anything better. The project name looks reasonable to me as well: `gpii-common-gpii-net`. Maybe someday we will buy `gpii-test.net` where we can stage changes to production DNS. 

Any other ideas on names for this env?

```
common/
  live/
    gpii-net/
      zone/
      dev/
        zone/
        alfredo/
        natarajaya/
        tyler/
      stg/
        zone/
      prd/
        zone/
```